fastlane/appstoreres/jetpack_metadata/source/keywords.txt

```diff
@@ -0,0 +1 @@
+social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal
```

Comment:
I'm pretty sure these were copied from the WordPress iOS app, right @leandroalonso @momo-ozawa ?

I'm not well versed in App Store Optimization, but I think it's redundant to include the app name in the keywords. Might be worth checking and updating for the next release.

Reply:
Yeah, looks like it was copied from the WPiOS app.

> I'm not well versed in App Store Optimization, but I think it's redundant to include the app name in the keywords. Might be worth checking and updating for the next release.

Just want to point out that our app name is "Jetpack: WP Security and Speed" -- not sure if that means we'd want to keep the "jetpack" keyword or not.

Reply:
The keywords weren't specified, so were copied. IMHO those are all still applicable, though we could probably focus in more on JP-specific things. Also IMHO "jetpack" is an important keyword to have because we don't have the "Jetpack" name all by itself